### DISCUSSION TITLE
How to finetune or continue training your models in google colab?

### DISCUSSION THREAD

**Discussion by pivolan** [coqui-ai/TTS#2454] [General Q&A]
I found only one place where placed information about how to start finetuning exists models, but it is only for console usage with ENV parameters: ![image](https://user-images.githubusercontent.com/903550/227369885-39c01009-ed7f-42b6-97d2-735801723f1e.png)
But no examples how to do this in code, or if I use google colab:

```
from TTS.config import load_config
...

# force the convertion of the custom characters to a config attribute
config.from_dict(config.to_dict())
config = load_config('config.json')
...

 init model
model = Vits(config, ap, tokenizer)
model.load_checkpoint(config=config,checkpoint_path='/content/checkpoints/checkpoint_1000.pth')

# init the trainer and 🚀
trainer = Trainer(
    TrainerArgs(), config, output_path, model=model, train_samples=train_samples, eval_samples=eval_samples
)

```

![image](https://user-images.githubusercontent.com/903550/227370461-4e44394f-ff58-4e3c-b051-f3122a47d1f6.png)

![image](https://user-images.githubusercontent.com/903550/227370557-69d1f8eb-0c53-4ae8-bb35-507d38e0cea8.png)

may be it will help somebody

**Comment by Thlenzi** [comment]
How to run the code above?
